### ISSUE TITLE
I keep getting "Local API Error: invalid continuation array" when attempting to load more results with Invidious API switched on

### ISSUE THREAD

**Issue by rjdg14** [FreeTubeApp/FreeTube#1900]
I use Freetube with the Invidious API turned on, and am finding that often when I attempt to load more search results, I get an "invalid continuation array" error instantly, with the results refusing to load (this also mentions "Local API", despite the fact that I've got it set to just use the Invidious API, with the fallback option disabled). I'm finding that when first loading Freetube, the problem is not usually there, but it begins after loading a video and then exiting back to the search results page. This seems like a bug, and it would be nice to get a fix as quickly as possible. Changing the Invidious API instance to a different URL seems to make no difference. Given that some results that I've already loaded will reload in the search results if I reenter the same search term, I suspect it may be a cache related issue.

I want to add in addition that I've still been finding buffering issues on some videos with the latest release - some videos have been taking 30 seconds to a minute before they will play via Invidious (which bypasses age restriction and geoblock gates, though this delay has been occuring randomly and most of these videos are unrestricted), and on some the buffer will freeze, typically after 5-10 minutes or so, meaning I have to exit and reenter the video for the buffer to continue loading, and I've never had Freetube behave like that before.

**Comment by ChunkyProgrammer** [comment]
do you still have this issue? I am unable to reproduce @rjdg14

**Comment by rjdg14** [comment]
Yep, I still have it. It is not the case initially but I can give you the instructions of how to reproduce it:
*Type in a search term (example: Donald Trump) with an Invidious proxy enabled (I'm using the US based https://inv.riverside.rocks/ proxy) and the Fallback setting disabled
*The initial results will load. Clicking the "fetch more results" button at this stage will work fine.
*Click on a video. Once it loads, click the back button which will return you to the results.
*Click on the "fetch more results" button now, and you will (or at least I do) get the "Local API Error: invalid continuation array" error. If you modify your search term slightly to load new results, the problem will not be there initially but will occur in the same manner once you've loaded a video and then return to that search term. If you want to load more results for an affected search term where at least one video has been watched, you will have to restart Freetube and then reenter the search term. Generally the best way I can overcome it is to load as many results as possible before watching a video after entering a new search term, since Freetube remembers these results when I hit the back button.

The problem is not present when Local API is enabled and Invidious disabled, but this means that age restricted videos will not load as they are behind an age gate in Europe (I have a verified Google account so can load them on YouTube via a European IP address but I still find the content warning a nuisance). A North American based Invidious proxy will load them minus description information.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@ChunkyProgrammer 

https://user-images.githubusercontent.com/73130443/168863781-6c2d3813-151a-4f21-b5ed-f15d2da949c8.mp4

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Ok so this error is gone but i do get this when i click on fetch more videos `There are no more results for this search`

**Comment by sysdbugfactory** [comment]
I have been affected for several months by the same issue: clicking the "fetch more results" does not fetch any results but raises this error: 

Local API error (Click to copy): TypeError: Cannot read properties of null (reading '1')

Using the AUR version from arch linux.

what kind of additional data do you need to help fix this bug ?
